extensions/spine/spine-assembler.js

```diff
@@ -296,6 +312,16 @@ var spineAssembler = {
         for (let index = 0, length = renderDatas.length; index < length; index++) {
             let data = renderDatas[index];
 
+            let key = data.material._hash;
+            let newKey = _updateKeyWithStencilRef(key, StencilManager.getStencilRef());
+            if (key !== newKey) {
```

Comment:
注释一下这里是为了对 mask 内的 skeleton data 区分 material

// For generate new material for skeleton render data nested in mask, otherwise skeleton inside/outside mask with same material will interfere each other

Reply:
查了一下，应该是材质球的通道没有clone造成的